src/code/graph.c

```diff
@@ -166,36 +174,41 @@ void Graph_TaskSet00(GraphicsContext* gfxCtx) {
         osSyncPrintf(VT_FGCOL(RED));
         osSyncPrintf("RCPが帰ってきませんでした。"); // "RCP did not return."
         osSyncPrintf(VT_RST);
+
         LogUtils_LogHexDump((void*)&HW_REG(SP_MEM_ADDR_REG, u32), 0x20);
         LogUtils_LogHexDump((void*)&DPC_START_REG, 0x20);
         LogUtils_LogHexDump(gGfxSPTaskYieldBuffer, sizeof(gGfxSPTaskYieldBuffer));
 
         SREG(6) = -1;
-        if (D_8012D260 != NULL) {
+        if (sPrevTaskWorkBuffer != NULL) {
             HREG(80) = 7;
             HREG(81) = 1;
             HREG(83) = 2;
-            D_8012D260 = D_8012D260;
-            Graph_DisassembleUCode(D_8012D260);
+            sPrevTaskWorkBuffer = sPrevTaskWorkBuffer;
+            Graph_DisassembleUCode(sPrevTaskWorkBuffer);
         }
         Fault_AddHungupAndCrashImpl("RCP is HUNG UP!!", "Oh! MY GOD!!");
     }
 
     osRecvMesg(&gfxCtx->queue, &msg, OS_MESG_NOBLOCK);
 
-    D_8012D260 = gfxCtx->workBuffer;
+    sPrevTaskWorkBuffer = gfxCtx->workBuffer;
     if (gfxCtx->callback != NULL) {
         gfxCtx->callback(gfxCtx, gfxCtx->callbackParam);
     }
 
-    time = osGetTime();
-    if (D_8016A550 != 0) {
-        D_8016A558 = (D_8016A558 + time) - D_8016A550;
-        D_8016A550 = time;
+    timeNow = osGetTime();
+    if (gAudioThreadUpdateTimeStart != 0) {
+        // The audio thread update is running
+        // Add the time already spent to the accumulator and leave the rest for the next cycle
+
+        gAudioThreadUpdateTimeAcc = gAudioThreadUpdateTimeAcc + timeNow - gAudioThreadUpdateTimeStart;
```

Comment:
Looks like this can be written with `+=` if the pad variable at the top is also removed. I suggest doing it since it makes nicer and consistent with the other `gAudioThreadUpdateTimeAcc +=` in audio
```suggestion
        gAudioThreadUpdateTimeAcc += timeNow - gAudioThreadUpdateTimeStart;
```